q2_annotate/busco/busco.py

```diff
@@ -140,6 +154,24 @@ def _evaluate_busco(
     common_args = _process_common_input_params(
         processing_func=_parse_busco_params, params=kwargs
     )
+    if isinstance(mags, MultiMAGSequencesDirFmt):
+        busco_results = _busco_helper(mags, common_args, additional_metrics)
+        busco_results["unbinned_contigs"] = pd.NA
+        busco_results["unbinned_contigs_count"] = pd.NA
+        for unbinned_id, unbinned_path in unbinned_contigs.sample_dict().items():
+            binned_dir = mags.path / unbinned_id
+            binned_fasta_paths = _get_fasta_files_from_dir(binned_dir)
```

Comment:
Instead of doing this you should use the `sample_dict` method available on MultiMAGSequencesDirFmt to retrieve all the required fasta files.